assets/js/admin/admin.js

```diff
@@ -221,4 +221,60 @@
 		$(this).append(video);
 
 	});
+
+	// Change span with file name when user selects a file
+	$('#everest-forms-import').change(function(e){
+		var file = $('#everest-forms-import').prop('files')[0];
+		$('#import-file-name').html(file.name);
+	});
+
+	$('.everest-forms-export-form-action').on('click', function() {
+		var form_id = $(this).closest('.everest-forms-export-form').find('#everest-forms-form-export').val();
+		if( 'undefined' === typeof form_id || 0 == form_id ) {
+			$(this).closest('.everest-forms-export-form').prepend('<div id="message" class="updated inline everest-froms-import_notice"><p><strong>Please select a form which you want to export.</strong></p></div>');
+			return false;
+		} else {
+			$(this).closest('.everest-forms-export-form').find('#message').remove();
+		}
+	});
+
+
+	$('.everest_forms_import_action').on('click', function () {
+		var file_data = $('#everest-forms-import').prop('files')[0],
+			form_data = new FormData();
+
+		form_data.append('jsonfile', file_data);
+		form_data.append('action', 'everest_forms_import_form_action');
+		form_data.append('security', everest_forms_admin.ajax_import_nonce);
+
+		$.ajax({
+			url: evf_email_params.ajax_url,
+			dataType: 'json',  // what to expect back from the PHP script, if anything
+			cache: false,
+			contentType: false,
+			processData: false,
+			data: form_data,
+			type: 'POST',
+			beforeSend: function () {
+				var spinner = '<i class="evf-loading evf-loading-active"></i>';
+				$('.everest_forms_import_action').closest('.everest_forms_import_action').append(spinner);
+				$('.everest-froms-import_notice').remove();
+			},
+			complete: function (response) {
+				var message_string = '';
+
+				$('.everest_forms_import_action').closest('.everest_forms_import_action').find('.evf-loading').remove();
+				$('.everest-froms-import_notice').remove();
+
+				if (response.responseJSON.success === true) {
```

Comment:
Follow yoda ruleset although jshint is not checked currently in the entire project.

Reply:
Now honors yoda styling, made appropriate changes.

Reply:
The code conforms to yoda ruleset now.